pkg/networkserver/downlink.go

```diff
@@ -851,13 +851,11 @@ func (ns *NetworkServer) processDownlinkTask(ctx context.Context) error {
 							})
 						}
 					} else {
-						if len(dev.RecentUplinks) == 0 {
-							return nil, nil, errUplinkNotFound
```

Comment:
Don't we need this? If the downlink doesn't contain the paths, then we need a recent uplink in order to determine the path, and can't do anything without it, right?

Reply:
I removed it because the `downlinkPathsFromRecentUplinks()` did not guarantee to return paths; so this check was not water tight. In any case, scheduling fails if there are no paths, but an early exit is nice. I added the check after.

All types of scheduling are now either skipped or error when there are no paths.